extra/innochecksum.cc

```diff
@@ -1288,6 +1303,9 @@ innochecksum_get_one_option(
 	case 'l':
 		is_log_enabled = true;
 		break;
+	case 'r':
+		skip_freed_pages = true;
+		break;
```

Comment:
Are these assignments really needed? If yes, what is the purpose of `&skip_freed_pages` in the parameter declaration?

Also, `argument MY_ATTRIBUTE((unused))` could be removed from the function definition.